src/common/pa_front.c

```diff
@@ -154,6 +154,7 @@ static PaUtilHostApiRepresentation **hostApis_ = 0;
 static int hostApisCount_ = 0;
 static int defaultHostApiIndex_ = 0;
 static int initializationCount_ = 0;
+static volatile int initializing_ = 0;
```

Comment:
What is the rationale for marking this as `volatile`? If there is a good reason, I think that there should be a comment explaining it. If there is no good reason, remove it.

Reply:
I original wanted to use an atomic here to have the  the feature also save when used from different threads. Unfortunately c98 has no interoperable way for this. volatile should work in practice, but it is implementation defined and there is no guarantee. I will just remove it, because my original use case is solved without it.

Reply:
Done